dask/array/tests/test_cupy.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pytest
+
+import dask.array as da
+from dask.array.utils import assert_eq
+
+cupy = pytest.importorskip('cupy')
+
+
+functions = [
+    lambda x: x,
+    pytest.mark.xfail(lambda x: da.expm1(x), reason="cupy doesn't support expm1"),
```

Comment:
http://docs-cupy.chainer.org/en/stable/reference/generated/cupy.expm1.html#cupy.expm1 ?

Reply:
Ah, thanks.  The real reason was that `np.expm1` isn't a real ufunc.  I've updated the `reason=` message.